changelog.md

```diff
@@ -1,3 +1,83 @@
+  * inline keyword: Added recompose strategy that topologically reconnects subdivided shapes.
+* New operations:
+  * modify operation.
+* Changes to existing features:
+  * split operation:
+     * When the first face is split away the face used to close hollow geometries becomes the new first face. This helps to preserve the first edge of the geometry.
+     * Previously, extruded geometries were only scaled and not split. Now, an actual split is performed. As a consequence, the resulting face and vertex indices and the first edges of faces can change. This can have an effect on the aligned scope or the result of probabilistic functions after a subsequent component split.
+  * offset operation: Improved performance. This slightly changed numerical accuracy.
+  * The initialization phase (attr and const evaluation) has been sped up, resulting in improved dynamic import performance.
+* Bugfixes:
+  * split operation: Fixed a bug where splitting extruded shapes led to a wrong result.
+  * extrude operation: Fixed a bug where existing vertex tags would not be propagated to the new side edges when extruding edge shapes.
+  * convexify operation:
+    * Fixed a bug where faces were not convexified at holes with only 3 vertices.
+    * Fixed a bug where zero-length edges were constructed.
+  * offset operation: Fixed a bug where a positive offset did not create holes correctly.
+  * inline(unify), union, subtract, intersect operations: Improved robustness when combining shapes with overlapping holes.
+ 
+## Built-In Codecs
+* Alembic and VUE encoders:
+  * Deprecated, they will be removed in the future. 
+
+# CITYENGINE SDK SDK 3.3.11351 CHANGELOG
+
+This section lists changes compared to CityEngine SDK 3.3.11173.
+
+## General Info
+* The focus of this release is to fix some issues which are important for certain client applications such as ArcGIS Pro. There is no CityEngine version using this version of the SDK.
+
+## PRT API
+* No changes.
+
+## PRTX API
+* No changes.
+
+## CGA
+* New annotations:
+    * @RuleFileDescription annotation.
+* Bugfixes:
+  * geometry.nVertices function: Fixed incorrect return value when applied to a shape with face vertices (created with comp(fv)).
+  * comp operation, comp function:
+    * comp(v): Fixed incorrect number and wrong order of components when applied to a shape with face vertices (created with comp(fv)).
+    * Fixed numerically unstable selectors in rare cases.
+  * cleanupGeometry operation: Fixed a rare crash for meshes with holes and multiple groups.
+  * split operation: Fixed a bug where duplicate vertices could be created when splitting edge shapes created with comp(fe).
+  * Improved numerical accuracy of scope calculation.
+
+## Built-In Codecs
+* USD decoder:
+  * Made it more stable: it now ignores problematic meshes instead of ignoring the whole asset.
+  * Fixed a bug that the USD importer ignored preview materials. 
+  * Fixed wrong texture resolve map keys from USDZ assets.
```

Comment:
I assume that this comes from the ticket name/description from CE-13059.
```suggestion
```
From my understanding the original reproduction of the ticket was user error resp. wrong assumptions on PRT behavior.
But we found several other smaller issues while debugging.
I assume that this comes from the ticket name/description from CE-13059.

But we are missing this line:
```suggestion
Fixed a bug that the USD Decoder wrongly percent-encoded keys of referenced assets when no ResolveMap was provided which lead to missing textures.
```

Reply:
removed the resolve map entry
added the other 2